### PR TITLE
docs: Update online detection doc and fiddle

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
#### Description of Change

This PR updates the online/offline how to guide. It simplifies the example to a single renderer-process based on, since the main-process example was only re-teaching IPC channels.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

cc: @erickzhao @molant

### CODE REVIEW

File: docs/tutorial/online-offline-events.md [hunk 1]
```diff
@@ -21,98 +21,70 @@ status of Electron, you should develop additional means for this check.
...
-```html
-<script src="renderer.js"></script>
-```
-
-and add the `renderer.js` file:
-
-```javascript fiddle='docs/fiddles/features/online-detection/renderer'
-const alertOnlineStatus = () => { window.alert(navigator.onLine ? 'online' : 'offline') }
+```js title='renderer.js'
+function onlineStatusIndicator () {
```
Comment (erickzhao) [inline docs/tutorial/online-offline-events.md:45]:
```suggestion
function updateOnlineStatus () {
```

nit: would prefer a verb for the function here

File: docs/tutorial/online-offline-events.md [hunk 2]
```diff
@@ -21,98 +21,70 @@ status of Electron, you should develop additional means for this check.
...
-npm start
+After launching the Electron application, you should see the notification:
 
-> electron@1.0.0 start /electron
-> electron .
+![Connection status](../images/connection-status.png)
 
-online
-```
+> If you need to communicate the connection status to the main process, use the [IPC renderer](../api/ipc-renderer.md) API.
```
Comment (erickzhao) [inline docs/tutorial/online-offline-events.md:90]:
```suggestion
> Note: If you need to communicate the connection status to the main process, use the [IPC renderer](../api/ipc-renderer.md) API.
```

File: docs/tutorial/online-offline-events.md [hunk 3]
```diff
@@ -21,98 +21,70 @@ status of Electron, you should develop additional means for this check.
...
-```html
-<script src="renderer.js"></script>
-```
-
-and add the `renderer.js` file:
-
-```javascript fiddle='docs/fiddles/features/online-detection/renderer'
-const alertOnlineStatus = () => { window.alert(navigator.onLine ? 'online' : 'offline') }
+```js title='renderer.js'
+function updateOnlineStatus () {
```
Comment (tonyferrell) [inline docs/tutorial/online-offline-events.md:45]: Function name doesn't match the call below.

Comment (tonyferrell) [reply docs/tutorial/online-offline-events.md:45]: Note: If Erick accepts my PR you can add the fiddle link back and have this focus on `renderer.js`.

File: docs/tutorial/online-offline-events.md [hunk 4]
```diff
@@ -21,98 +21,70 @@ status of Electron, you should develop additional means for this check.
...
 
-alertOnlineStatus()
+onlineStatusIndicator()
 ```
 
-After launching the Electron application, you should see the notification:
-
-![Online-offline-event detection](../images/online-event-detection.png)
-
-### Event detection in the Main process
```
Comment (tonyferrell) [inline docs/tutorial/online-offline-events.md:63]: Did we intentionally remove forwarding the event to the main process from this document? Perhaps we could add a brief description if we don't want it (and maybe a link to the IPC documentation?)

Comment (tonyferrell) [reply docs/tutorial/online-offline-events.md:63]: Ah, just noticed your description discussing this. Might be worth to leave some bread-crumbs here for folks who aren't familiar with all concepts?

File: docs/tutorial/online-offline-events.md [hunk 5]
```diff
@@ -21,98 +21,70 @@ status of Electron, you should develop additional means for this check.
...
-```javascript fiddle='docs/fiddles/features/online-detection/renderer'
-const alertOnlineStatus = () => { window.alert(navigator.onLine ? 'online' : 'offline') }
+```js title='renderer.js'
+function updateOnlineStatus () {
+  document.getElementById('status').innerHTML = navigator.onLine ? 'online' : 'offline'
+}
 
-window.addEventListener('online', alertOnlineStatus)
-window.addEventListener('offline', alertOnlineStatus)
+window.addEventListener('online', onlineStatusIndicator)
```
Comment (Ethan-Arrowood) [inline docs/tutorial/online-offline-events.md:49]:
```suggestion
window.addEventListener('online', updateOnlineStatus)
```

File: docs/tutorial/online-offline-events.md [hunk 6]
```diff
@@ -21,98 +21,70 @@ status of Electron, you should develop additional means for this check.
...
+window.addEventListener('offline', onlineStatusIndicator)
```
Comment (Ethan-Arrowood) [inline docs/tutorial/online-offline-events.md:50]:
```suggestion
window.addEventListener('offline', updateOnlineStatus)
```

File: docs/tutorial/online-offline-events.md [hunk 7]
```diff
@@ -21,98 +21,70 @@ status of Electron, you should develop additional means for this check.
...
 
-alertOnlineStatus()
+onlineStatusIndicator()
```
Comment (Ethan-Arrowood) [inline docs/tutorial/online-offline-events.md:52]:
```suggestion
updateOnlineStatus()
```

File: docs/tutorial/online-offline-events.md (hunk 4 above)
Comment (Ethan-Arrowood) [reply docs/tutorial/online-offline-events.md:63]: There is a footnote at the end of the file with a link to docs for utilizing IPC channels

**Review by zcbenz — APPROVED** [review]

**Pull request merged**